README.md

```diff
@@ -4,18 +4,40 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 ![npm](https://img.shields.io/npm/v/electron-ngrx?style=flat-square)
 
-**ElectronNgrx** is a library for dispatching NGRX actions between different Electron windows on a multi-window Electron Apps.
+## Motivation
+Most everyone who has worked with multiple Electron windows knows that it requires tons of repetitive coding. 
 
-The library supports the following API methods:
+The ones who tried to get to multiple windows to communicate a decent amount of information between them, then go and basically have to end up building a rest API.
+….No one wants that.
 
-`dispatchToParent` - Dispatch NGRX action data through the window's parent.
+### The solution
+ElectronNgrx offers an easy to use solution. It seamlessly dispatches actions and selects data from state's across multiple windows.
 
-`dispatchToId`  - Dispatch NGRX action data to a specific Electron window by providing the destination window's ID.
+##Usge:
+electronNgrx returns an Angular service with these functions
 
-`dispatchToRoute`  - Dispatch NGRX action data to a specific Route providing the destination route's details.
+`dispatchToParent` - Dispatch NGRX action to the window's parent.
 
+`dispatchToId` - Dispatch NGRX action to a specific Electron window by providing the destination window ID.
 
+`dispatchToRoute` - Dispatch NGRX action to all window’s on the specific route.
 
-## Build
+`selectFromId` - select data from the state of a different window with the window ID provided.  
```

Comment:
`selectFromId` - select data from the state of the window that matches the given id